public/css/st-tailwind.css

```diff
@@ -328,6 +336,14 @@
     padding: 0;
 }
 
+.padding1 {
```

Comment:
Wait, what?.. 1 = 2 = 0?

Reply:
The magic of copy&paste :/
I can fix that.. or remove the tailwind classes alltogether if you don't wanna continue building on those :D

Reply:
If you didn't even notice it, I guess it's good to be gone

Reply:
They were leftovers anyway, not used. Think I changed it to properties on existing classes anyway.